src/js/containers/aboutTheData/AgencyDetailsContainer.jsx

```diff
@@ -144,6 +144,7 @@ const AgencyDetailsContainer = ({ modalClick, agencyName, agencyCode }) => {
                 setRows(parseRows(res.data.results));
                 setTotalItems(res.data.page_metadata.total);
                 setLoading(false);
+                setError(false);
```

Comment:
We should be setting the error to false before the request before `setLoading(true);`.

e.g. If a user makes a request, and visually sees data in the table, then sorts, we make another request and the user visually sees an error, then another user makes a new request the error will still be shown while the request is in flight where we would want to visually notify the user with a loading screen that we are retrieving more data.